### ISSUE TITLE
Add time waited to first in sigrid room queue.

### ISSUE THREAD

**Issue by ludde127** [bjornregnell/sigrid#38]
Add a timer that shows some relevant information about who has waited the longest to allow ambulances to know which rooms are most congested time wise. One example of a solution is to add the time waited for the first person in the queue (Longest waiting).

**Comment by bjornregnell** [comment]
First step here: https://github.com/bjornregnell/sigrid/commit/b0a40ccd98f89b55f4d2343342563f4ffb39efbe

**Comment by ludde127** [comment]
I have also been working on a solution and have one (probably finnished) in this fork https://github.com/ludde127/sigrid .

**Comment by bjornregnell** [comment]
Yeah. Sorry for not checking with you before starting working on this.
My solution is simpler I think. But there is one method left with ??? and there is some clutter in the printout...

**Comment by ludde127** [comment]
No problem on not checking with me, I should have written here when i started the work. I think my solution works and the print out is quite nice, but i am a bit scared about the changes to User. 
![image](https://user-images.githubusercontent.com/59414524/200627834-1c6c7aef-72e1-4a48-aabc-b5a1c88419e8.png)

**Comment by bjornregnell** [comment]
Perhaps you can build on my work by integrating your printing functions and implement `def longestWaitingTimeMinutes: Int = ???` in Room and use that in your strings? And I'll close your current PR?

**Comment by bjornregnell** [comment]
Or make one waitingtime method per quee def longestHelpWaitingMinutes  etc

**Comment by bjornregnell** [comment]
Or do you want me to finish the printing inspired by your PR?

**Comment by ludde127** [comment]
I added the nicer printing to showRoomLong, not sure what showRoomShort is used for as showRoomLong seams to be used in beppe, sigrid, and sigrid monitor. I have tested adding multiple users to queues and if one is removed the next ones timer is shown, unsure what more to test. I settled on this look. 
![image](https://user-images.githubusercontent.com/59414524/200647569-aae7d9ea-058b-4b39-9bfd-f3918faef19a.png)

**Comment by bjornregnell** [comment]
Nice! PR welcome! (I can review it tomorrow afternoon the earliest.)

**Comment by bjornregnell** [comment]
Perhaps remove decimals in minutes?

**Comment by ludde127** [comment]
> 

Yes that seams lika a nice idea, will probably only clutter the printout in read use! Creating a PR.

**Comment by bjornregnell** [comment]
Thing left to do: Add longest queuing time in room overview at login and sigrid monitor. I think it was there i used the short form (?)

**Comment by ludde127** [comment]
I have started working on adding the maximum queuing time of both queues in each room to the short form. 
At the moment the login page looks like this.
![image](https://user-images.githubusercontent.com/59414524/201480471-df7e3e03-c0ed-40d9-8ca0-5fbe1e94f980.png)

**Comment by bjornregnell** [comment]
Nice! I suggest skipping the dot and blank, so just `Maxkötid` in one word to save space.

**Comment by bjornregnell** [comment]
I actually meant just the current time of the student in front just like in the long form, but maximum queueing time is also interesting as it says something about the peak load during the 2h time slot. Perhaps both figures shoudl be in the overview of active rooms as it can be a basis for students to go to a room with low load.

**Comment by bjornregnell** [comment]
So my suggestion is having both `Köat x min` and `Maxkötid y min`, where the latter is counted during the lates 2h slot eith 8-10, 10-12, 13-15 or 15-17. Or did you mean the max of the two queues `köat`? That might also be a good definition of the maximum queuing-time momentarily,  (The maximum peak load is perhaps not so important).
Let's discuss this live.

**Comment by ludde127** [comment]
I meant the current maximum queuing time of anyone in any of the queues (not the maximum over some timeframe). I don't really think that the maximum queuing time over a 2h time slot is useful for the login page. Yes, lets discuss live!

**Comment by bjornregnell** [comment]
Ok I see. Yes, that's probably more useful as an overview. (Statistics over history is actually another feature.) Go ahead with the PR inline your original intent, and I can take a look at the code when you have a PR, if you like.
